CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+0.9.0
+---
+* **command line**
+  - implement 1200-baud UART reset
```

Comment:
i'd say "implement 1200-baud UART bootloader reset when flashing boards that support it"